akka-docs/src/main/paradox/typed/interaction-patterns.md

```diff
@@ -116,17 +163,23 @@ The function is running in the receiving actor and can safely access state of it
 
 ## 1:1 Request-Response with ask from outside the ActorSystem
 
-In an interaction where there is a 1:1 mapping between a request and a response we can use @scala[`ActorRef.?` implicitly provided by `akka.actor.typed.scaladsl.AskPattern`]@java[`akka.actor.typed.javadsl.AskPattern.ask`] to send a message to an actor and get a @scala[`Future[Response]`]@java[`CompletionState[Response]`] back.
+Some times you need to interact with actors from outside of the actor system, this can be done with fire-and-forget as described above or through another version of `ask` that returns a @scala[`Future[Response]`]@java[`CompletionStage<Response>`] that is either completed with a succesful response or failed with a `TimeoutException` if there was no response within the specified timeout. 
+ 
+To do this we use @scala[`ActorRef.?` implicitly provided by `akka.actor.typed.scaladsl.AskPattern`]@java[`akka.actor.typed.javadsl.AskPattern.ask`] to send a message to an actor and get a @scala[`Future[Response]`]@java[`CompletionState[Response]`] back.
```

Comment:
Perhaps for completeness (and not only showing symbolic operator without a name)
`ActorRef.ask` (or `ActorRef.?`)